index.html

```diff
@@ -9195,6 +9195,16 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     [[!WEBRTC10]], as well as to enable the WebRTC 1.0 API to be implemented on top of
     the ORTC API, utilizing a Javascript "shim" library. This section discusses WebRTC
     1.0 compatibility issues that have been encountered by ORTC API implementers.</p>
+    <section id="spelling*">
+      <h3>Spelling differences</h3>
+      <p>WebRTC 1.0 supports the <code>replaceTrack</code> method and the
+      <code>RTCDTMFSender</code> interface, whereas ORTC supports the <code>setTrack</code>
+      method and the <code>RTCDtmfSender</code> interface. In order to provide
+      backward compatibility, ORTC implementations should allow calls to the
+      <code>replaceTrack</code> method to invoke <code>setTrack</code>, and
```

Comment:
Can you make this part of the IDL? This way tooling can notice that the spec requires something that isn't in an implementation, and we're currently building such tooling for Chromium.